### PR TITLE
Code Polishing (Sonar)

### PR DESCRIPTION
**Pull request by garyrussell**
- remove redundant modifiers
- overridable methods called from ctors

### CODE REVIEW

File: spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/ConnectionFactoryUtils.java [hunk 1]
```diff
@@ -25,6 +23,8 @@
 import org.springframework.transaction.support.TransactionSynchronizationManager;
 import org.springframework.util.Assert;
 
+import com.rabbitmq.client.Channel;
```
Comment (artembilan) [inline spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/ConnectionFactoryUtils.java]: Looks like you have different import config for AMQP project.
Or am I missing anything here again?
imports

**Comment by artembilan** [comment]
Everything else looks good.
Just let me know about that epic import issue.

File: spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/ConnectionFactoryUtils.java (hunk 1 above)
Comment (garyrussell) [inline spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/ConnectionFactoryUtils.java]: I haven't changed anything in my STS; I have always had these settings, which (I think) I got from Mark:

screen shot 2015-12-08 at 10 12 25 am

Comment (artembilan) [inline spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/ConnectionFactoryUtils.java]: OK. Doing the same settings and merging...
Good morning, BTW!

**Pull request merged**